cfme/tests/infrastructure/test_provisioning.py

```diff
@@ -40,6 +40,33 @@ def vm_name():
     return vm_name
 
 
+@pytest.fixture(scope="function")
+def provision_request(provisioning, provider, request, vm_name):
+    # generate_tests makes sure these have values
+    template, host, datastore = map(provisioning.get, ('template', 'host', 'datastore'))
+
+    provisioning_data = {
+        'vm_name': vm_name,
+        'host_name': {'name': [host]},
+        'datastore_name': {'name': [datastore]},
+        'num_vms': "1",
+    }
+
+    if provider.type == 'rhevm':
+        provisioning_data['provision_type'] = 'Native Clone'
+    elif provider.type == 'virtualcenter':
+        provisioning_data['provision_type'] = 'VMware'
```

Comment:
@izapolsk other provisioning types are tested separately(e.g. pxe), this just makes sure that we have correct provision type selected